test/model/relationalProperties.test.ts

```diff
@@ -0,0 +1,52 @@
+import { manyOf, primaryKey } from '../../src'
+import { Database } from '../../src/db/Database'
+import { InternalEntityProperty, Relation } from '../../src/glossary'
+import { defineRelationalProperties } from '../../src/model/defineRelationalProperties'
+
+it('returns an enumerable relation property', () => {
+  const dictionary = {
+    user: {
+      id: primaryKey(String),
+      posts: manyOf('post'),
+    },
+    post: {
+      id: primaryKey(String),
+      title: String,
+    },
+  }
+
+  const db = new Database(dictionary)
+
+  db.create('post', {
+    [InternalEntityProperty.primaryKey]: 'id',
+    [InternalEntityProperty.type]: 'post',
+    id: 'post-1',
+    title: 'First Post',
+  })
+  db.create('post', {
```

Comment:
Do you think it'd be a good idea to use `oneOf` relation just to reduce the number of entities we create in the test? It should result in the same behavior from the utility's perspective.

Reply:
Yeah, that's a good point. Originally I actually had more tests but decided to pare it down, because it was sort of duplicating the other relational tests. Will rewrite to be a bit shorter perhaps.